frontend/src/concepts/areas/const.ts

```diff
@@ -66,4 +66,8 @@ export const SupportedAreasStateMap: SupportedAreasState = {
     featureFlags: ['disableDistributedWorkloads'],
     requiredComponents: [StackComponent.KUEUE],
   },
+  [SupportedArea.MODEL_REGISTRY]: {
+    featureFlags: ['disableModelRegistry'],
+    requiredComponents: [StackComponent.TRUSTY_AI], // TODO: Change to model-registry once the component is available in DSC
```

Comment:
This is just a workaround in place, if not it's gonna be really hard to test the feature until is enabled by the operator.

cc @zdtsw any idea on how to hack a cluster to show an unavailable dsc component?